src/controllers/api/2014/classController.ts

```diff
@@ -10,6 +11,21 @@ import SimpleController from '@/controllers/simpleController';
 import Spell from '@/models/2014/spell';
 import Subclass from '@/models/2014/subclass';
 
+// --- Zod Schemas ---
+const ParamsSchema = z.object({
+  index: z.string().min(1),
+});
+
+const LevelParamsSchema = z.object({
+  index: z.string().min(1),
+  level: z.coerce.number().int().min(1).max(20),
+});
```

Comment:
anywhere we share properties we can do `merge` so we dont have to copy-paste the same props everywhere
```ts
const LevelParamsSchema = ParamsSchema.merge(
  z.object({
    level: z.coerce.number().int().min(1).max(20),
  })
)
```

Reply:
Instead, I consolidated to a single schema file.